typescript/ecs/fargate-service-with-local-image/local-image/app.py

```diff
@@ -11,5 +11,5 @@ def hello():
     return html.format(name=os.getenv("NAME", "world"), hostname=socket.gethostname())
 
 if __name__ == "__main__":
-    app.run(host='0.0.0.0', port=80)
+    app.synth()(host='0.0.0.0', port=80)
```

Comment:
Are the empty parens correct before the args?

Reply:
Actually looks like it was an accidental python replace for Fargate local-image. 
Reverting to "run()".